tools/codegen/dest/__init__.py

```diff
@@ -1,5 +1,5 @@
 from .lazy_ir import LazyIR as LazyIR
-from .lazy_ir import gen_lazy_shape_inference_decl as gen_lazy_shape_inference_decl
+from .lazy_ir import GenLazyShapeInferenceDefinition as GenLazyShapeInferenceDefinition
```

Comment:
Just curious, why `as` is needed here?
```suggestion
from .lazy_ir import GenLazyShapeInferenceDefinition
```

Reply:
im not really sure. it was that way for all the others and i didn't think about it.  i'll leave it this way for this PR but could change them all at once in a pr to core if it works.